src/Symfony/Bundle/FrameworkBundle/Command/AboutCommand.php

```diff
@@ -26,7 +27,7 @@
  *
  * @final since version 3.4
```

Comment:
if there is a use case for extending it and we want to support this use case, we must remove `@final`, to allow extending.